charts/telepresence-oss/values.schema.yaml

```
@@ -0,0 +1,580 @@
+    $ref: "#/$defs/logLevel"
+
+  managerRbac:
+    type: object
+    additionalProperties: false
+    properties:
+      create:
+        description: Create RBAC resources for traffic-manager with this release
+        type: boolean
+      namespaces:
+        description: Declares a fixed set of managed namespaces
+        deprecated: true
+        type: array
+        items:
+          $ref: "#/$defs/rfc1123Label"
+
+  maxNamespaceSpecificWatchers:
+    description: >-
+      Threshold controlling when the traffic-manager switches from using watchers for each managed namespace to using
+      cluster-wide watchers
+    type: integer
+    minimum: 0
+    maximum: 50
+
+  nameOverride:
+    description: Override the default name prefix used when creating RBAC resources
+    type: string
+
+  namespaces:
+    description: Declares a fixed set of managed namespaces. Mutually exclusive to namespaceSelector
+    type: array
+    items:
+      $ref: "#/$defs/rfc1123Label"
+
+  namespaceSelector:
+    description: Declares the managed namespace using matchLabels and matchExpressions. Mutually exclusive to namespaces
+    $ref: "#/$defs/labelSelector"
+
+  nodeSelector:
+    description: Define which Nodes you want to the Traffic Manager to be deployed to.
+    $ref: "#/$defs/nodeSelector"
+
+  podAnnotations:
+    description: Annotations for the Traffic Manager Pod
+    type: object
+    additionalProperties:
+      type: string
+
+  podCIDRs:
+    description: podCIDRs is the verbatim list of CIDRs used when the podCIDRStrategy is set to environment
+    type: array
+    items:
+      type: string
+
+  podCIDRStrategy:
+    description: Define the strategy that the traffic-manager uses to discover what CIDRs the cluster uses for pods
+    type: string
+    enum:
+      - auto
+      - coverPodIPs
+      - environment
+      - nodePodCIDRs
+
+  podLabels:
+    description: Labels for the Traffic Manager Pod
+    type: object
+    additionalProperties:
+      type: string
+
+  podSecurityContext:
+    description: The Kubernetes SecurityContext for the traffic-manager Pod
+    $ref: "#/$defs/podSecurityContext"
+
+  priorityClassName:
+    description: Name of the existing pod priority class to be used
+    type: string
+
+  prometheus:
+    type: object
+    additionalProperties: false
+    properties:
+      port:
+        description: Set this port number to enable a prometheus metrics http server for the traffic manager
+        type: integer
+
+  rbac:
+    type: object
+    additionalProperties: false
+    properties:
+      only:
+        description: Only create the RBAC resources and omit the traffic-manger
+        type: boolean
+
+  readinessProbe:
+    description: Define readinessProbe for the Traffic Manger.
+    $ref: "#/$defs/probe"
+
+  replicaCount:
+    description: Number or replicas for the traffic-manager. The Traffic Manager only support running with one replica at the moment.
+    type: integer
+    minimum: 1
+    maximum: 1
```

```suggestion
    const: 1
```
we could use `const` unless there's some tooling that doesn't support that keyword.

---

Heh, I need to refresh my json-schema skills :-)